migrations/2022-06-21-123144_language-tags/up.sql

```diff
@@ -20,6 +20,15 @@ INSERT INTO
 LANGUAGE (id, code, name)
     VALUES (0, 'und', 'Undetermined');
 
+ALTER TABLE LANGUAGE
+    ALTER COLUMN id DROP DEFAULT;
+
+DROP SEQUENCE language_id_seq;
+
+ALTER TABLE LANGUAGE
+    ALTER COLUMN id
+    ADD GENERATED ALWAYS AS IDENTITY;
```

Comment:
In this case id = 0 is hardcoded to mean "unknown language" so it needs to be inserted specifically. And these alter commands are really what we would have to run in a new migration for every single id column instead of rewriting existing migrations.